styles/lib/_modal.css

```diff
@@ -0,0 +1,55 @@
+.modal{
+    display: none;
+    position: fixed;
+    z-index: 100;
+    padding-top: 12rem;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    overflow: auto;
+    background-color: rgba(0,0,0,0.4);
+}
+
+.modal-content{
+    position: relative;
+    background-color: var(--white);
+    margin: auto;
+    padding: 0;
+    width: 45%;
+    -webkit-animation-name: animatetop;
+    -webkit-animation-duration: 0.4s;
+    animation-name: animatetop;
+    animation-duration: 0.4s
+}
+
+
+.modal-header{
+    padding: 1.2rem 2rem;
+    border-bottom: thin solid var(--grey);
+    margin: 0.8rem 0;
```

Comment:
for values less than 10px try to use px instead of rem

Reply:
Ohk, I'll update this.